server/map_rpc_server_test.go

```diff
@@ -128,6 +142,84 @@ func TestGetSignedMapRoot(t *testing.T) {
 	}
 }
 
+func TestGetSignedMapRootByRevision(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	ctx := context.Background()
+
+	tests := []struct {
+		desc string
+		req *trillian.GetSignedMapRootByRevisionRequest
+		mapRoot trillian.SignedMapRoot
+		snapShErr, lsmrErr error
+		wantErr bool
```

Comment:
is `wantErr` redundant / equal to ` snapShErr != nill || lsmrErr != nil`?

Reply:
Yes, effectively.  I had it in the test tables explicitly to make it clear what outcome the test expects.